src/lib/header/Header.svelte

```diff
@@ -12,8 +13,7 @@
 	{#if $user.walletAddress}
 		<div class="relative cursor-pointer" on:mouseenter={() => (showDisconnect = true)}>
 			<div class="px-2 lonk pretty-lonk">
-				Connected as {$user.walletENSAddress ||
-					`${$user.walletAddress.slice(0, 2)}...${$user.walletAddress.slice(-4)}`}
+				Connected as {$user.walletENSAddress || prettyAddress($user.walletAddress)}
```

Comment:
Would you mind reverting this? Have a change here in the another pr

Reply:
The whole function is just this file?